frontend/src/app/commonComponents/Dropdown.jsx

```diff
@@ -2,13 +2,27 @@ import React from "react";
 import PropTypes from "prop-types";
 import { v4 as uuidv4 } from "uuid";
 
-const Dropdown = ({ options, label, name, onChange, selectedValue }) => {
+const Dropdown = ({
```

Comment:
I made some changes to this one on my branch as well, so we'll need to reconcile them. I like the idea of having a default value and "-select-" though, I wanted this too!